src/main/java/com/sforce/cd/apexUnit/client/QueryConstructor.java

```diff
@@ -271,14 +275,14 @@ public static String getApexClassInfo(String apexClassId) {
 	public static String getApexTriggerInfo(String apexClassId) {
 		String soql = "";
 		if (apexClassId != null && !apexClassId.equals("")) {
-			soql = "SELECT Id,Name,ApiVersion,LengthWithoutComments FROM ApexTrigger where Id = '"
-					+ escapeSingleQuote(apexClassId) + "'";
+			soql = "SELECT Id,Name,ApiVersion,LengthWithoutComments FROM ApexTrigger where Id = "
+					+ escapeSingleQuote(apexClassId);
```

Comment:
@adarsh-ramakrishna-sfdc  The query response to 
*SELECT Id,Name,ApiVersion,LengthWithoutComments FROM ApexTrigger where Id = null*
returns zero objects.